vendor/lowrisc_ip/dv_utils/dv_utils.core

```diff
@@ -0,0 +1,23 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:dv_utils"
+description: "DV utilities"
+
+filesets:
+  files_dv:
+    depend:
+      - lowrisc:dv:common_ifs
+      - lowrisc:prim:assert:0.1
+    files:
+      - top_pkg.sv
```

Comment:
Has this file been modified? That's the file at the given revision in opentitan: https://github.com/lowRISC/opentitan/blob/c383de06/hw/dv/sv/dv_utils/dv_utils.core

If you want to do modifications you need to add a patch and setup patching in `vendor/lowrisc_dv_utils.lock.hjson`. You can find documentation for that process at https://docs.opentitan.org/doc/ug/vendor_hw/#how-to-add-patches-on-top-of-the-imported-code.

In the same patch, also add the top_pkg.sv file that you currently add as separate file into the `vendor/lowrisc_ip/dv_utils/` directory. If you don't add `top_pkg.sv` as part of a patch it will be overwritten with the next run of the vendor tool.

(That being said, the easier option might be to add a core named `lowrisc:constants:top_pkg` without patching anything.)

Reply:
Sure, I'll add the top_pkg core file to icache/dv/ to prevent having to patch anything.